### PR TITLE
CB-22031 remove rpm -q check from install_metering_rpm_manually state and update heartbeat version to latest

### PR DESCRIPTION
**Pull request by sodre90**

### CODE REVIEW

File: core/src/main/resources/application.yml [hunk 1]
```diff
@@ -813,7 +813,7 @@ existing-stack-patcher:
     metering-azure-metadata:
       enabled: false
       date-before: '2022-01-18'
-      custom-rpm-url: https://archive.cloudera.com/cp_clients/thunderhead-metering-heartbeat-application-0.1-SNAPSHOT.x86_64.rpm
+      custom-rpm-url: https://archive.cloudera.com/cp_clients/thunderhead-metering-heartbeat-application-1.0.0-b8780.x86_64.rpm
```
Comment (lnardai) [inline core/src/main/resources/application.yml:816]: What's this version based on exactly? Why this version?

Comment (sodre90) [reply core/src/main/resources/application.yml:816]: it is based on the cloudbreak-images repo

File: orchestrator-salt/src/main/resources/salt/salt/metering/init.sls [hunk 2]
```diff
@@ -14,7 +14,7 @@
 install_metering_rpm_manually:
   cmd.run:
     - name: "rpm -i {{ metering_rpm_location }}"
-    - onlyif: "! rpm -q {{ metering_package_name }}"
+    - onlyif: "! test -e /opt/metering-heartbeat"
```
Comment (lnardai) [inline orchestrator-salt/src/main/resources/salt/salt/metering/init.sls:17]: This is more error prone if the directory ever get's moved.
Can't we rather check for systemd unit file or systemd service name?

Comment (sodre90) [reply orchestrator-salt/src/main/resources/salt/salt/metering/init.sls:17]: I think we should remove the whole installation

**Comment by keyki** [comment]
Closing the PR, please re-open if it's still relevant.

**Pull request closed without merging**